### PR TITLE
docs: add electron 23

### PR DESCRIPTION
**Pull request by sriting**

### CODE REVIEW

File: blog/electron-23-0.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    url: 'https://github.com/georgexu99'
+    image_url: 'https://github.com/georgexu99.png?size=96'
+  - 
+    name: erickzhao
+    url: 'https://github.com/erickzhao'
+    image_url: 'https://github.com/erickzhao.png?size=96'
+slug: electron-23-0
+---
+
+Electron 23.0.0 已发布！ 此次升级中包含 Chromium `110`, V8 `11.0`, 和 Node.js `18.12.1`。  此外，取消了对Windows 7/8.1的支持。 请阅读下文了解更多详情！
```
Comment (BlackHole1) [inline blog/electron-23-0.md]:
```suggestion
Electron 23.0.0 已发布！ 此次升级中包含 Chromium `110`, V8 `11.0`, 和 Node.js `18.12.1`。  此外，取消了对 Windows 7/8.1 的支持。 请阅读下文了解更多详情！
```

File: blog/electron-23-0.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+---
+
+Electron 团队很高兴发布了 Electron 23.0.0！ 您可以通过 `npm install electron@latest` 进行安装，或者从我们的 [发布网站](https://releases.electronjs.org/releases/stable) 下载它。 继续阅读此版本的详细信息。
+
+如果您有任何反馈，请在Twitter上与我们分享，或加入我们的社区 [Discord](https://discord.com/invite/electronjs)！ Bug 和功能请求可以在 Electron 的 [问题跟踪器](https://github.com/electron/electron/issues) 中报告。
```
Comment (BlackHole1) [inline blog/electron-23-0.md]:
```suggestion
如果您有任何反馈，请在 Twitter 上与我们分享，或加入我们的社区 [Discord](https://discord.com/invite/electronjs)！ Bug 和功能请求可以在 Electron 的 [问题跟踪器](https://github.com/electron/electron/issues) 中报告。
```

File: blog/electron-23-0.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## 重要变化
+
+### 架构（Stack）更新
+
+* Chromium `110`
+    * [Chromium有新版本110](https://developer.chrome.com/blog/new-in-chrome-110/)
+    * [Chromium有新版本109](https://developer.chrome.com/blog/new-in-chrome-109/)
```
Comment (BlackHole1) [inline blog/electron-23-0.md]:
```suggestion
    * [Chromium 有新版本110](https://developer.chrome.com/blog/new-in-chrome-110/)
    * [Chromium 有新版本109](https://developer.chrome.com/blog/new-in-chrome-109/)
```

File: blog/electron-23-0.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+* 将 `label` 属性添加到 [`Display`](https://www.electronjs.org/docs/latest/api/structures/display) 对象上。 [#36933](https://github.com/electron/electron/pull/36933)
+* 添加了 `app.getPreferredSystemLanguages()` API 以返回用户的系统语言。 [#36035](https://github.com/electron/electron/pull/36035)
+* 添加了对 [WebUSB](https://developer.mozilla.org/en-US/docs/Web/API/WebUSB_API) API 的支持。 [#36289](https://github.com/electron/electron/pull/36289)
+* 添加了对 [`SerialPort.forget()`](https://developer.mozilla.org/en-US/docs/Web/API/SerialPort/forget) 的支持，以及当给定来源被撤销时在 [Session](https://www.electronjs.org/docs/latest/api/session) 对象上发出的新事件 `serial-port-revoked` 。 [#35310](https://github.com/electron/electron/pull/35310)
+* 添加了新的 `win.setHiddenInMissionControl` API 以允许开发者选择退出 macOS 上的任务控制。 [#36092](https://github.com/electron/electron/pull/36092)
+
+## 放弃对 Windows 7/8.1 支持
+
+Electron 23 不再支持Windows 7/8.1。 Electron 遵循计划中的 Chromium 弃用政策，该政策将 [在 Chromium 109 中弃用 Windows 7/8/8.1 支持（在此处阅读更多信息）](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en)。
```
Comment (BlackHole1) [inline blog/electron-23-0.md]:
```suggestion
Electron 23 不再支持 Windows 7/8.1。 Electron 遵循计划中的 Chromium 弃用政策，该政策将 [在 Chromium 109 中弃用 Windows 7/8/8.1 支持（在此处阅读更多信息）](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en)。
```

File: blog/electron-23-0.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
++     scrollTouchEdge()
++   } else if (event.type === 'gestureScrollEnd') {
++     scrollTouchEnd()
++   }
++})
+```
+
+## 终止对 20.x.y 的支持
+
+根据项目的[支持政策](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy)，Electron 20.x.y 已经达到了支持的终点。 Developers and applications are encouraged to upgrade to a newer version of Electron.
```
Comment (BlackHole1) [inline blog/electron-23-0.md]: "Developers and applications are encouraged to upgrade to a newer version of Electron."

没有翻译

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

**Review by Cyberhan123 — APPROVED** [review]

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**